app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -15,7 +15,7 @@ class Publishers::Vacancies::DocumentsController < Publishers::Vacancies::BaseCo
                               application/vnd.openxmlformats-officedocument.presentationml.presentation ].freeze
   FILE_SIZE_LIMIT = 10.megabytes
 
-  helper_method :form
+  helper_method :form, :step
```

Comment:
This makes a private method public, as far as I can tell. Maybe that private method should be public now?

Again, I can't see where it's being used.

Reply:
I think it's a hangover from another approach I took. I'll remove it.